app/javascript/chat/draw/index.jsx

```diff
@@ -107,44 +121,47 @@ function Draw({ sendCanvasImage }) {
     <div className="p-4 grid gap-2 crayons-card mb-4 connect-draw">
       <div className="mb-1 draw-title">
         <h2>Connect Draw</h2>
-        <div className="colors" style="pointer-events: all;">
-          {palettes.map((color) => (
+        <div className="connect-draw__colors">
+          {chatDrawPalettes.map((color) => (
             <button
-              className="color"
+              className="connect-draw__color"
               onClick={handleChangeColor}
               style={`background-color: ${color}`}
-              title={`color-${color}`}
+              title={`Change color to ${color}`}
+              key={`id-${color}`}
```

Comment:
non-blocking: the key can be just color.

```suggestion
              key={color}
```